components/Accordion/__tests__/Accordion.test.tsx

```diff
@@ -1,20 +1,29 @@
 import { fireEvent, render } from '@testing-library/react';
-
 import {
   ACCORDION_CONTENT,
   ACCORDION_TOGGLE_BUTTON,
   SCREEN_READER_ONLY,
 } from 'common/constants/testIDs';
-import { Default } from '../__stories__/Accordion.stories';
 import { toggleMessages } from '../../ScreenReaderOnly/ScreenReaderOnly';
+import Accordion from '../Accordion';
 
 describe('Accordion', () => {
   it('should render invisible text that turns visible on toggle click', async () => {
-    const component = render(<Default {...Default.args} />);
```

Comment:
Why does this no longer work? It's quite nice to share Storybook code with test code, wouldn't you say?

Reply:
TS was giving me an error saying that `<Default />` was not a JSX element. But, I think I got a work around for it.